SessionMessagingKit/Sending & Receiving/MessageSender.swift

```diff
@@ -597,7 +597,8 @@ public final class MessageSender {
                 // real message has no use when we delete a message. It is OK to let it be.
                 try interaction.with(
                     serverHash: message.serverHash,
-                    
+                    // Update the authorId if blinded id is enabled, otherwise it will remain the same
```

Comment:
Should we be setting the `authorId` to the `blindedId` when initially creating the `Interaction` instead of changing it after the fact? (just wondering if this could result in further unexpected behaviours down the line)

We even have both values cached on the `threadData` - `(self.viewModel.threadData.currentUserBlindedPublicKey ?? self.viewModel.threadData.currentUserPublicKey)`